src/main/scala/org/ergoplatform/network/ModeFeature.scala

```diff
@@ -0,0 +1,65 @@
+package org.ergoplatform.network
+
+import org.ergoplatform.nodeView.state.StateType
+import org.ergoplatform.settings.{NodeConfigurationSettings, PeerFeatureIds}
+import scorex.core.network.PeerFeature
+import scorex.core.network.PeerFeature.Id
+import scorex.core.serialization.ScorexSerializer
+import scorex.util.serialization.{Reader, Writer}
+
+
+case class ModeFeature(stateType: StateType,
```

Comment:
Let's add class comments.
More general question - why do we use one feature with 4 params, instead of 4 independent features?